transmogrifier/sources/xml/marc.py

```diff
@@ -153,6 +153,17 @@ def create_subfield_value_string_from_datafield(
             Marc.create_subfield_value_list_from_datafield(xml_element, subfield_codes)
         )
 
+    @staticmethod
+    def concatenate_subfield_value_strings_from_datafield(
+        source_record: Tag, tag: str, subfield_codes: str
+    ) -> str:
+        return " ".join(
+            Marc.create_subfield_value_string_from_datafield(
```

Comment:
I'm hesitant to request more changes on this PR, as this is kind of minor, but I might propose:
  * making this a `@classmethod`
  * having it then call `cls.create_subfield_value_string_from_datafield(...)`

I think unless there is good reason, having a `staticmethod` call that class that it's defined on is probably a good indication it could/should be a `@classmethod`.

Reply:
That said, nice change!  I wouldn't be surprised if other parts of the MARC record could potentially use this...

Reply:
Ah, good point, I agree `@classmethod` would be better

Reply:
Ah, thanks for the heads up! Updated. 😄

Reply:
Looks good!